### DISCUSSION TITLE
void-function `defcustom-lsp`

### DISCUSSION THREAD

**Discussion by jwennerstrum1** [emacs-lsp/lsp-mode#3531] [General]
Pull #3405 replaced `defcustom` with `defcustom-lsp` in `lsp-generate-settings.el`, however I do not see a definition for this function anywhere.  I just tried downloading `lsp-haskell`, however loading this file fails due to `defcustom-lsp` being undefined.  Is this function defined in a separate package that I am missing?  @yyoncho could you please provide some context for this change?

Thanks!

**Comment by seagle0128** [comment]
`defcustom-lsp` is a macro in `lsp-mode.el` (Line 8026).

**Reply by jwennerstrum1** [reply]
Aha.  Yup, refreshing lsp-mode did the trick.  Sorry I did not catch that earlier!
